ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -1,11 +1,11 @@
 # Defaults file for the compatibility layer role.
 ---
-eessi_version: "2021.12"
+eessi_version: "2022.11"
 
 custom_overlays:
   - name: eessi
     source: git
-    url: https://github.com/EESSI/gentoo-overlay.git
+    url: https://github.com/trz42/gentoo-overlay.git
```

Comment:
Just pointing this out, so we don't forget to revert this back to `EESSI` after https://github.com/EESSI/gentoo-overlay/pull/84 is merged...

Reply:
To answer my own question: yes, you've just updated the `main` branch in your fork with https://github.com/EESSI/gentoo-overlay/pull/84, OK.